app/globals.css

```diff
@@ -12,3 +12,12 @@ html {
   background-color: #1e1e1e;
   color: #ffffff;
 }
+
+:root {
+  --font-geist-sans: "Share Tech", sans-serif;
+  --font-geist-mono: "Share Tech Mono", monospace;
+}
+
+body {
+  font-family: var(--font-geist-sans);
+}
```

Comment:
I wonder if this is necessary, considering that `Geist` didn't require this to be set up as the default font. `app/layout.tsx` might be the only file required to be changed, although I haven't tested this yet.

Reply:
Due to the root, it's necessary. If I tried to remove, it won't work. But if your is worked without it, then that's fine. I will remove that